rom/rom.go

```diff
@@ -328,8 +328,16 @@ func getImage(url string, p string, w uint) error {
 	if err != nil {
 		return err
 	}
-	if w > 0 && uint(img.Bounds().Dx()) > w {
-		img = resize.Resize(w, 0, img, resize.Bilinear)
+	if w > 0 {
+		if uint(img.Bounds().Dx()) >= uint(img.Bounds().Dy()) {
```

Comment:
OK. Didn't know.
Actually I added that line when I had issues to compile locally.
Because I thought my changes did not worked.